src/com/nutrons/steamworks/RobotBootstrapper.java

```diff
@@ -211,6 +214,15 @@ private Command hopperDrive(double distance1, double angle, double distance2) {
         Command.parallel(RobotBootstrapper.this.climbtake.pulse(true).delayFinish(500, TimeUnit.MILLISECONDS).then(RobotBootstrapper.this.climbtake.pulse(false).delayFinish(500, TimeUnit.MILLISECONDS)),
             Command.serial(RobotBootstrapper.this.drivetrain.driveDistance(distance1, 1, 10).endsWhen(Flowable.timer(1300, TimeUnit.MILLISECONDS), true),
                 RobotBootstrapper.this.drivetrain.turn(angle, 10),
-                RobotBootstrapper.this.drivetrain.driveDistance(distance2, 1, 10).endsWhen(Flowable.timer(1300, TimeUnit.MILLISECONDS), true)));
+
+                Command.parallel(
+                    RobotBootstrapper.this.turret.automagicMode().delayFinish(13000, TimeUnit.MILLISECONDS),
+                    RobotBootstrapper.this.shooter.auto().delayStart(1000, TimeUnit.MILLISECONDS),
```

Comment:
We changed this to `auto()` from `pulse()`. Is this guaranteed to work / whats the difference?